libraries/AP_IOMCU/AP_IOMCU.cpp

```diff
@@ -892,12 +892,20 @@ void AP_IOMCU::bind_dsm(uint8_t mode)
   setup for mixing. This allows fixed wing aircraft to fly in manual
   mode if the FMU dies
  */
-bool AP_IOMCU::setup_mixing(RCMapper *rcmap, int8_t override_chan,
+bool AP_IOMCU::setup_mixing(const RC_Channel *channel_roll,
+                            const RC_Channel *channel_pitch,
+                            const RC_Channel *channel_throttle,
+                            const RC_Channel *channel_yaw,
+                            int8_t override_chan,
                             float mixing_gain, uint16_t manual_rc_mask)
 {
     if (!is_chibios_backend) {
         return false;
     }
+    if (channel_roll == nullptr || channel_pitch == nullptr || channel_throttle == nullptr || channel_yaw == nullptr) {
+        return false;
```

Comment:
it is not OK to just return false here. That means no mixing at all if any single channel is missing. Disabling mixing that uses a channel that is missing is OK, but disabling all mixing for one missing channel isn't.

Reply:
Fixed.